### ISSUE TITLE
Code Table Request - new part preservation: other (with removal of "unknown")

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#7010]
### Goal
_Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

Provide flexibility in recording preservations

### Context
_Describe why this new value is necessary and existing values are not._

Some preservation values are not used enough to justify a new preservation in the code table. This would provide a way to record them without making the code table over complicated. The value "unknown" exists, but does not reflect the fact that a preservation is known, just not "standard". Rather than add a second version of "unknown" to the code table, @dustymc has proposed that unknown be removed from the code table and for all parts using that preservation be replaced with preservation = other and method = unknown.

### Table
_Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation

### Proposed Value
_Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

other

### Proposed Definition
_Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

Use this preservation type when the preservation is not included in the code table and there are not yet good reasons for requesting a new preservation term. Describe the actual preservation in the attribute method. If it is desirable to record preservation unknown, then use this term and place unknown in the preservation method.

### Collection type
_Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

N/A

#### Attribute data type
_If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_

N/A

#### Attribute controlled values
_If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate._

N/A

#### Attribute units
_if numerical values should be accompanied by units, provide a link to the appropriate units table._

N/A

#### Part preservation attribute affect on "tissueness"
_if a new part preservation is requested, please add the affect it would have on "tissueness": No Influence, Allows, or Denies_

No influence? It will depend upon the actual preservation, so this seems like the best choice?

### Priority
_Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

Needed for incoming collection OGL

### Example Data
_Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

Here are a sampling of potential use from the incoming OGL genomics collection.

preservation | preservation method 
-- | --
other | Davidson's fix
other | 250 mM EDTA, 3% NaCl
other | Stored in GSCN
other | Fixed in 4% PFA in seawater for 3 hours.
other | stored in either no preservative or Davidson’s fixative
other | Treatment: 0.05 ml of 0.1 M NaCitrate
other | Preserved for microbiology: SBM homogenization buffer
other | GSCN buffer
other | SBM homogenization buffer
other | Fixed in 4% PFA in seawater for overnight. Transferred to 70% ethanol.
other | Fixed in 4% PFA in seawater on ice for 4 hours. Transferred to 70%   ethanol.
other | Fixed in 2.5% glutaraldehyde in marPHEM buffer at 4°C for 3 hours.   Transferred to marPHEM buffer.
other | Fixed in 4% PFA in seawater for 1 hour. Transferred to 70% ethanol.
other | Fixed in 4% PFA in seawater overnight. Transferred to 70% ethanol.
other | Fixed in 4% PFA in seawater for 3 hours. Transferred to 70% ethanol.
other | Fixed in 4% PFA in seawater for 4 hours. Transferred to 70% ethanol.
other | Homogenized in 100 µM ectoine in SBM homogenization buffer, pH   7.0.Homogenate was restreak onto SBM plates.
other | Fixed in 4% PFA in seawater overnight in ice bath. Transferred to 70%   ethanol.
other | Shell was cracked to allow better fixative penetration. Fixed in 2.5%   glutaraldehyde in marPHEM buffer at 4°C overnight. Transferred to marPHEM   buffer.
other | Shell was cracked to allow better fixative penetration. Fixed in 4% PFA   in seawater for 3 hours. Transferred to 70% ethanol.
other | fixed in 4% PFA in seawater for 4 hours on ice. Shells were not cracked.
other | Homogenized in 100 µM ectoine in SBM homogenization buffer, pH 7.0.   Diluted further to 10 to the -1 using the same buffer. Diluted homogenate was   restreak onto SBM plates.
other | Treatment: 0.2 ml of 0.1 M NaCitrate
other | bleach
other | GD
other | 0.05 ml of 0.1 M NaCitrate
other | GSCN
other | 250mM EDTA + 3% NaCl
other | Homogenized in 100 µM ectoine in SBM homogenization buffer, pH 7.0
other | DNAgard
other | homogenized in 1ml 75% filter sterilized seawater
other | homogenized
other | SBM liquid
other | Homogenized in SBM+ homogenization buffer for microbiology
other | Homogenized in SBM homogenization buffer with 100mM ectoine for   microbiology
other | Homogenized in SBM homogenization buffer, pH 7 with 100mM ectoine for   microbiology
other | Biofilm scrapped from surface of wood that was directly streaked onto a   plate
other | 1 mL SBM homogenization buffer, transferred to 3X glyceron stocks, stored   in -80
other | For microbiology, preserved in SBM homoginization buffer with eutoine,   then 0.5 ml 40% glycerol + 0.5 ml homogenate, then to -80C
other | Treatment: 50% 0.25M EDTA pH 10, 47.5% Ethanol
other | Treatment: 2.5% gluteraldehyde-MarPHEM Buffer to MarPHEM, fixed for 15   hours in 4*C. For EM.
other | Treatment is bleach
other | Preserved in 0.25mM EDTA, 10mM Tris pH 10
other | 0.1 ml of 0.1 M NaCitrate
other | unknown amount of 0.1 M NaCitrate
other | 1.00, could be 0.1 ml of 0.1 M NaCitrate
other | 0.2 ml of 0.1 M NaCitrate
other | 0.25M EDTA pH10 20% PEG 8000
other | 0.25M EDTA pH10 3.5% NaCl
other | 0.25M EDTA pH10 5% PEG 8000
other | 0.25M EDTA pH10 saturated NaCl
other | 1/4 replicate microbank stock of Kuphus isolate 2141T, Passage IV in STNB   media
other | 2.5% GA into 1% ozmium into 70% ethanol
other | 2/4 replicate microbank stock of Kuphus isolate 2141T, Passage IV in STNB   media
other | 20% PEG 8000
other | 3/4 replicate microbank stock of Kuphus isolate 2141T, Passage IV in STNB   media
other | 4% PFA > 70% ethanol
other | 4/4 replicate microbank stock of Kuphus isolate 2141T, Passage IV in STNB   media
other | 5% PEG 8000 saturated NaCl
other | Chemically fixed 1 hr after return from    boat.
other | Chemically fixed 1 hr after return from    boat. Sample treatment listed as DES
other | Chemically fixed 1 hr after return from boat.
other | Chemically fixed 1 hr after return from boat. Sample treatment listed as   DES
other | Chemically fixed immediately after return from site (1.5-hr?).
other | Chemically fixed immediately after return from site (1.5-hr?). Sample   treatment listed as DES
other | Chemically fixed on 1 hr after return from boat.
other | Chemically fixed on 1 hr after return from boat. Sample treatment listed   as DES
other | Chemically fixed on boat.
other | Chemically fixed on boat. Sample treatment listed as DES
other | Chemically fixed within1 hr after return from collection.
other | Chemically fixed within1 hr after return from collection. Sample   treatment listed as DES
other | Collected Monday, processed Monday. Chemical fixes done on boat.
other | Collected Monday, processed Monday. Chemical fixes done on boat. Sample   treatment listed as DES
other | Collected Sunday, processed Tues. Too old?
other | Collected Sunday, processed Tues. Too old?
other | Collected Sunday, processed Tues. Too old?  Sample treatment listed as DES
other | Dissected and smeared on an agar plate and freezer stocks were made.
other | Dissected the day of death on a bed of ice
other | Davidson's Fix +C1939
other | Shell was cracked to allow better fixative penetration. Fixed with 2.5%   glutaraldehyde, 2.0% paraformaldehyde, 0.1 M Na-cacodylate buffer, pH 7.2 in   seawater for 3 hrs on ice. Now in 70% ethanol.
other | Shell was cracked to allow better fixative penetration. Fixed with 4.0%   paraformaldehyde in seawater for 3 hrs on ice. Now in 70% ethanol.
other | Fixed in 2.5% glutaraldehyde in 1.5X marPHEM buffer, pH 7.4 with 9%   sucrose at 4°C for 15 hours. Now in 1.5X marPHEM buffer, pH 7.4 with 9%   sucrose.
other | Fixed on Boat
other | Fixed on Boat Sample treatment listed as DES
other | Fixed on shipboard.
other | Fixed on shipboard. Sample treatment listed as DES
other | Homogenate in 40% gycerol at 1:1 ratio given sample(s).
other | SBM homogenization buffer and actono, 40% glycol.
other | SBM homogenization buffer w/ ectoine.
other | SBM homogenization buffer.
other | Remainder of gill in 40% glycerol, 1:1 ratio, dry ice.
other | Fixative GD
other | Homogenized in 0.5 ml SBM homogenization buffer, pH 7.0 with 100 µm   ectoine
other | Homogenized in 1.0 ml SBM homogenization buffer, pH 7.0 with 100 µm   ectoine
other | Homogenized in 100 µM ectoine in SBM homogenization buffer, pH 7.0.
other | Davidson’s fix
other | Clark
other | DES
other | Preserved for microbiology
other | did not preserve.
other | did not preserve. Sample treatment listed as DES
other | 0.25M EDTA + saturated salt
other | 20%DMSO + 0.25M EDTA
other | 4M GTC
other | DEW
other | Tissue was homogenized in SBM homogenization buffer, pH 7.0 with 100 µM   ectoine.
other | Preserved in 0.25 M EDTA, pH 10.0 with 47.5% Ethanol.
other | Homogenized in SBM homogenization buffer, pH 7.0 with 100 µM ectoine
other | Preserved in 0.25 M EDATA, pH 10.0 with 47.5% Ethanol.
other | TA (10mM Tris pH8.0, 1mM EDTA)
other | TA buffer (10mM Tris pH 8, 1mM EDTA)
other | 62.7% EtOH and 0.13M EDTA
other | 62.7%EtOH+0.13M EDTA
other | EDTA .25M pH 10, EtOH 47.5%
other | EDTA 0.25M pH10, EtOH 47.5%
other | EDTA 0.25M pH11
other | EDTA 0.25M pH12
other | EDTA 0.25M pH13
other | EDTA 0.25M pH14
other | EDTA 0.25M pH15
other | EDTA 0.25M pH16
other | EDTA 0.25M pH17
other | EDTA 0.25M pH18
other | EDTA 0.25M pH19
other | Trizol

### Available for Public View
_Most data are by default publicly available. Describe any necessary access restrictions._

yes

### Helpful Actions

- [x] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [x] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators @happiah-madson @falco-rk 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [x] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [x] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by dustymc** [comment]
I don't think that both this is 'unknown' should exist. Recommend moving forward with this and updating existing unknown to 'other' method 'unknown.' (Maybe better as a separate Issue, @Jegelewicz plz guide!)

**Comment by Jegelewicz** [comment]
> Maybe better as a separate Issue, @Jegelewicz plz guide!

These need to be done together, so I think keeping them in one issue makes sense. Original request will be modified to reflect the changes.

**Comment by campmlc** [comment]
I support this also, but with some comments. Many of these are complex procedures involving both fixation and preservation methods that already exist in the preservation code table values, such as ethanol EDTA.  Should ethanol,  EDTA etc be entered as additional attribute vales? How searchable can we make these? Can a search on EDTA in part preservation discover these records?

**Comment by Jegelewicz** [comment]
> Should ethanol, EDTA etc be entered as additional attribute vales?

I am planning to do that as much as I can possibly figure out.

> Can a search on EDTA in part preservation discover these records?

Short answer is no under "normal" search conditions, but I think they could still be found with "Part search" - assuming it doesn't time out...
![image](https://github.com/ArctosDB/arctos/assets/5725767/c4706690-bf42-4294-ab2a-b5568e2d724a)

**Comment by Jegelewicz** [comment]
> how would unknowns be handled? By saying "other" with unknown in remarks? Or by not listing a part preservation? (I think part of the reason that we added Unknown to our drop down was that GGBN required it).

The proposal is "other" with unknown in remarks.

**Comment by happiah-madson** [comment]
That sounds great as long as I don't have to figure out how that goes to GGBN 😬

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This sounds like a good proposal. I have checked a box.

**Comment by Jegelewicz** [comment]
other has been added to the code table

@dustymc can you facilitate the transition of "unknown" to other?
